receiver/mongodbreceiver/scraper.go

```diff
@@ -231,3 +294,48 @@ func serverAddressAndPort(serverStatus bson.M) (string, int64, error) {
 		return "", 0, fmt.Errorf("unexpected host format: %s", host)
 	}
 }
+
+func (s *mongodbScraper) findSecondaryHosts(ctx context.Context) ([]string, error) {
+	result, err := s.client.RunCommand(ctx, "admin", bson.M{"replSetGetStatus": 1})
+	if err != nil {
+		s.logger.Error("Failed to get replica set status", zap.Error(err))
+		return nil, fmt.Errorf("failed to get replica set status: %w", err)
+	}
+
+	members, ok := result["members"].(primitive.A)
+	if !ok {
+		return nil, fmt.Errorf("invalid members format")
```

Comment:
why use Errorf with no interpolation? Please consider adding a %v directive and log members, or use errors.New